aws/table_aws_ecs_task.go

```diff
@@ -18,16 +17,11 @@ func tableAwsEcsTask(_ context.Context) *plugin.Table {
 		Name:        "aws_ecs_task",
 		Description: "AWS ECS Task",
 		List: &plugin.ListConfig{
-			Hydrate:           listEcsTasks,
-			ParentHydrate:     listEcsClusters,
+			Hydrate:    listEcsTasks,
 			KeyColumns: []*plugin.KeyColumn{
-				{
-					Name:    "cluster_arn",
-					Require: plugin.Optional,
-				},
 				{
 					Name:    "cluster_name",
-					Require: plugin.Optional,
+					Require: plugin.Required,
```

Comment:
I'm not sure how I feel about requiring a cluster name when querying for tasks. I may want to find all tasks in my account that have over 6GB of memory. 

But in either case, the following doesn't work:
```
> select task_arn, started_at, stopped_at from aws_ecs_task where cluster_name in ('<redacted cluster 1>', '<redacted cluster 2>')
+----------+------------+------------+
| task_arn | started_at | stopped_at |
+----------+------------+------------+
+----------+------------+------------+
```

But this works when querying against `aws_ecs_cluster`:
```
> select title from aws_ecs_cluster where title in ('<redacted cluster 1>', '<redacted cluster 2>')
+-------------------------------------+
| title                               |
+-------------------------------------+
| <redacted cluster 1>                |
| <redacted cluster 2>                |
+-------------------------------------+
```

Reply:
@mildebrandt for this query below, I hope you have the associated tasks in it?
```
> select task_arn, started_at, stopped_at from aws_ecs_task where cluster_name in ('<redacted cluster 1>', '<redacted cluster 2>')```

```
> select task_arn, cluster_name, created_at from aws_ecs_task where cluster_name in ('test-ec2', 'cluster45')
+------------------------------------------------------------------------------------+--------------+---------------------+
| task_arn                                                                           | cluster_name | created_at          |
+------------------------------------------------------------------------------------+--------------+---------------------+
| arn:aws:ecs:us-east-1:632902152528:task/test-ec2/7b4feb2a7438451fa7616018fecd97fa  | test-ec2     | 2021-11-03 07:23:53 |
| arn:aws:ecs:us-east-1:632902152528:task/cluster45/b8057246631a4201995ac5f3e34c88ea | cluster45    | 2021-11-03 07:24:04 |
+------------------------------------------------------------------------------------+--------------+---------------------+
```

Reply:
Also cluster_name will not be a required column.

> select task_arn, cluster_name, created_at, cpu, memory from aws_ecs_task
```
+------------------------------------------------------------------------------------+--------------+---------------------+-----+--------+
| task_arn                                                                           | cluster_name | created_at          | cpu | memory |
+------------------------------------------------------------------------------------+--------------+---------------------+-----+--------+
| arn:aws:ecs:us-east-1: 6329021522343248:task/cluster45/b8057246631a4201995ac5f3e34c88ea | cluster45    | 2021-11-03 07:24:04 | 256 | 512    |
+------------------------------------------------------------------------------------+--------------+---------------------+-----+--------+
```

Reply:
@mildebrandt We have made some changes in the existing code, could you please take the latest pull and then try running your queries and let us if you are getting the expected result.

Reply:
All of the queries on my side look good. This is great! Thank you @c0d3r-arnab and @bigdatasourav!

Reply:
@mildebrandt thanks for your feedback, It will be available in today's release.